File: CS/dxSample/Form1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraEditors;

namespace dxSample {
    public partial class Form1 : XtraForm {
        public Form1() {
            InitializeComponent();
            InitLayoutControl();
        }
        private void InitLayoutControl() {
            customLayoutControl1.OptionsView.HighlightFocusedItem = true;
            InitItems();
        }
        private void InitItems() {
            layoutControlGroup1.AddItem("TestItem 0", new TextEdit());
            
            CustomLayoutControlItem item = (CustomLayoutControlItem)layoutControlGroup1.AddItem("TestItem 1 ", new TextEdit());
            item.HighlightFocusedItemEx = false;

            layoutControlGroup1.AddItem("TestItem 2 ", new TextEdit());
        }
    }
}

File: VB/dxSample/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraEditors

Namespace dxSample
    Partial Public Class Form1
        Inherits XtraForm

        Public Sub New()
            InitializeComponent()
            InitLayoutControl()
        End Sub
        Private Sub InitLayoutControl()
            customLayoutControl1.OptionsView.HighlightFocusedItem = True
            InitItems()
        End Sub
        Private Sub InitItems()
            layoutControlGroup1.AddItem("TestItem 0", New TextEdit())

            Dim item As CustomLayoutControlItem = CType(layoutControlGroup1.AddItem("TestItem 1 ", New TextEdit()), CustomLayoutControlItem)
            item.HighlightFocusedItemEx = False

            layoutControlGroup1.AddItem("TestItem 2 ", New TextEdit())
        End Sub
    End Class
End Namespace
